app-text/xlreader/xlreader-0.9.0.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Excel document reader"
+DESCRIPTION="Read data from Excel 97+ spread sheets without Microsoft! \
+Provides an API to write applications that read Excel documents. The xlreader \
+tool converts data to tab delimited, CSV or SQL inserts. Written in C. Based on \
+the the Java version by Andrew Khan."
+HOMEPAGE="http://giffin.org/xlreader"
+COPYRIGHT="2001 David Giffin"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/project/xlreader/xlreader/0.9.0/xlreader-0.9.0.tgz?r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fxlreader%2F&ts=1512775451&use_mirror=superb-dca2"
```

Comment:
Please remove useless stuff after `?`.

Reply:
You might wish to use SF's redirector service:
```
SOURCE_URI="https://downloads.sourceforge.net/xlreader/xlreader-$portVersion.tgz"
```